packages/amplify-cli/src/utils/post-install-initialization.ts

```diff
@@ -40,14 +43,19 @@ export const postInstallInitialization = async () => {
 const resolvePackageRoot = (packageName: string) => {
   const resolveDir = path.parse(require.resolve(packageName)).dir;
   const pathParts = resolveDir.split(path.sep);
-  return pathParts.slice(0, pathParts.indexOf(packageName) + 1).join(path.sep);
+  return pathParts.slice(0, pathParts.indexOf(descopePackageName(packageName)) + 1).join(path.sep);
```

Comment:
we were not handling the case where the package name is scoped i.e `@aws-amplify/amplify-opensearch-simulator`, in this logic to resolve the package root. 
These changes address that issue.